docs/basics/getting-started.md

```diff
@@ -16,3 +16,4 @@ Things here might include:
 ### Software
   * [docs.linuxserver.io](https://docs.linuxserver.io/)
   * [Docker](https://www.docker.com/) & [Docker Compose](https://docs.docker.com/compose/)
+  * [Plex](https://www.plex.tv/)
```

Comment:
This change should not be part of this PR

Reply:
no it should not, im not sure the best way to fix that..

Reply:
I could think of two options:
- remove the change in a new commit
- modify the commits and and force push them to rewrite the history (very ugly)

Reply:
ok the first one is what i was thinking but i wanted to make sure before i mess something up as im new to git. thanks.